### DISCUSSION TITLE
Support for sending text to HmIP-WRCD?

### DISCUSSION THREAD

**Discussion by fridomac** [SukramJ/aiohomematic#932] [Q&A] [answered]
Hello!
Is it possible to send text to the e-Ink-Display of the HmIP-WRCD?
Could not find anything about it (other than that the device is supported) either in this repo or the custom-hahhomatic one.
Thank you for the great project!
Bye, Frido.

**Comment by SukramJ** [comment, marked as answer]
See #176
